src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -2,11 +2,24 @@
 using Raven.Server.Config.Attributes;
 using Raven.Server.Config.Settings;
 using Sparrow;
+using Sparrow.Platform;
 
 namespace Raven.Server.Config.Categories
 {
     public class StorageConfiguration : ConfigurationCategory
     {
+        public StorageConfiguration()
+        {
+            // On Windows, DiscardVirtualMemory can result in high CPU usage due to contention on the
+            // PTE entry (in Win Server 2016 and Win Server 2019), we want to avoid it by default.
```

Comment:
This is specific to WS2016 and WS2019, right? But we're disabling it on all Windows versions. Is it really what we want?

Reply:
We saw that in those two, and we don't _need_ to have it. Better to have more consistent behavior than try to test across everything that exists.